### PR TITLE
Feature/social contact links

### LINKED ISSUE #105: Add Social and Contact Links to Site using Config values
AS a User
When I see the social links
I want to be able to click them and go to the correct place

AC:
https://www.facebook.com/RebuildingBlack
https://www.instagram.com/rebuildingblack/
https://twitter.com/rebuildingblack
Mailto: social@rebuildblackbusiness.com
Paypal: waiting on, will get by by 6/10

Area: https://d.pr/i/7tbKI6

https://trello.com/c/x7yKO3ce

### PR DESCRIPTION
**Pull request by juanri0s**
# Describe your PR

Related to #105
Fixes #105

## Pages/Interfaces that will change
About contact card link
Home Contact Us link
Footer social links minus PayPal (still waiting on this)

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1. Go to About page and  inspect/clickcontact card link
2. Go to Home page and  inspect/clickcontact us link
3. Go to Footer and inspect/click social links 

### Additional notes
https://trello.com/c/x7yKO3ce

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -1,114 +1,135 @@
...
-              heading="WHO WE ARE"
-              message={WHO_WE_ARE_MESSAGE}
-              marginTop="3rem"
-              marginBottom="1.9375rem"
-              dividerMargin="3.9375rem"
-            />
-          </Flex>
-        </Flex>
+    <ErrorBoundary>
+      <StaticQuery
```
Comment (magnificode) [inline src/pages/about.js:15]: Wondering if we can maybe wrap this static query around only the part of the markup that is leveraging the queried data?

My lack of Gatsby knowledge here is showing a little bit, but would feel nice from a markup perspective :D

File: src/pages/index.js [hunk 2]
```diff
@@ -1,190 +1,217 @@
...
-                Rebuild
-              </Text>
-              <Text
-                fontSize={['100px', '100px', '100px', '150px']}
-                fontWeight="900"
-                lineHeight="1"
-                fontFamily={theme.fonts['heading-slab']}
-                textTransform="uppercase"
+    <ErrorBoundary>
+      <StaticQuery
```
Comment (magnificode) [inline src/pages/index.js:20]: Same query question here, can we wrap only the dynamic bits of markup in this query?

File: src/pages/about.js (hunk 1 above)
Comment (juanri0s) [reply src/pages/about.js:15]: I will take a look, what you're saying makes sense

File: src/pages/index.js (hunk 2 above)
Comment (juanri0s) [reply src/pages/index.js]: should be fixed now.

File: src/pages/about.js (hunk 1 above)
Comment (juanri0s) [reply src/pages/about.js]: should be fixed now.

**Pull request merged**